api/commands/run.ts

```diff
@@ -15,13 +15,21 @@ const isSuccessfulAction = (actionResult: dataform.IActionResult) =>
   actionResult.status === dataform.ActionResult.ExecutionStatus.CACHE_SKIPPED ||
   actionResult.status === dataform.ActionResult.ExecutionStatus.DISABLED;
 
-export function run(graph: dataform.IExecutionGraph, dbadapter: dbadapters.IDbAdapter): Runner {
-  return Runner.create(dbadapter, graph).execute();
+export function run(
+  dbadapter: dbadapters.IDbAdapter,
+  graph: dataform.IExecutionGraph,
+  partiallyExecutedRunResult?: dataform.IRunResult
+): Runner {
+  return Runner.create(dbadapter, graph, partiallyExecutedRunResult).execute();
 }
 
 export class Runner {
-  public static create(dbadapter: dbadapters.IDbAdapter, graph: dataform.IExecutionGraph) {
-    return new Runner(dbadapter, graph);
+  public static create(
```

Comment:
you can delete this create method if you want, it adds very little value, my bad :p